server/src/session_manager.ts

```diff
@@ -110,17 +110,22 @@ export class SessionManager {
             proxy = `http://${proxy}`;
         }
 
+        let logProxyUrl: string = proxy;
```

Comment:
Technically this is more than just the URL (contains username/password/ip/port). Maybe `loggedProxy`?